docs/source/doctest_fixtures.py

```diff
@@ -117,7 +124,7 @@
 
 model = SimpleModel(num_channels, num_classes)
 
-optimizer = torch.optim.SGD(model.parameters(), lr=0.001)
+optimizer = DecoupledSGDW(model.parameters(), lr=0.001)
```

Comment:
why this change?

Reply:
Due to https://github.com/pytorch/pytorch/issues/133415. Stateless optimizer causes problems

Reply:
does that imply composer is broken with torch.optim.SGD?

Reply:
Currently, yes 😬. We have workaround though (decoupledSGDW) and hoping to get into torch2.4.1 for a fix